hack/lib/common.sh

```diff
@@ -510,51 +512,80 @@ function kube::common::start_controller_manager {
 
     CTLRMGR_LOG=${LOG_DIR}/kube-controller-manager.log
 
-    if [ "${IS_RESOURCE_PARTITION}" == "true" ]; then
-      KUBE_CONTROLLERS="nodelifecycle"
-
-      ${CONTROLPLANE_SUDO} "${GO_OUT}/hyperkube" kube-controller-manager \
-        --v="${LOG_LEVEL}" \
-        --allocate-node-cidrs="${KUBE_CONTROLLER_MANAGER_ALLOCATE_NODE_CIDR}" \
-        --cluster-cidr="${KUBE_CONTROLLER_MANAGER_CLUSTER_CIDR}" \
-        --vmodule="${LOG_SPEC}" \
-        --service-account-private-key-file="${SERVICE_ACCOUNT_KEY}" \
-        --root-ca-file="${ROOT_CA_FILE}" \
-        --cluster-signing-cert-file="${CLUSTER_SIGNING_CERT_FILE}" \
-        --cluster-signing-key-file="${CLUSTER_SIGNING_KEY_FILE}" \
-        --enable-hostpath-provisioner="${ENABLE_HOSTPATH_PROVISIONER}" \
-        ${node_cidr_args[@]+"${node_cidr_args[@]}"} \
-        --pvclaimbinder-sync-period="${CLAIM_BINDER_SYNC_PERIOD}" \
-        --feature-gates="${FEATURE_GATES}" \
-        "${cloud_config_arg[@]}" \
-        --kubeconfig "${kubeconfigfilepaths}" \
-        --controllers="${KUBE_CONTROLLERS}" \
-        --leader-elect=false \
-        --cert-dir="${CERT_DIR}" \
-        --default-network-template-path="${ARKTOS_NETWORK_TEMPLATE}" >"${CTLRMGR_LOG}" 2>&1 &
+    KUBE_CONTROLLERS="*"
+    if [[ "${IS_SCALE_OUT}" == "true" ]]; then
```

Comment:
since it only one line difference from scale-up and scale-out command lines, probably it will be clearer to use the if-else block to make the desired args for the --controllers arg to avoid such dup for most the command line

Reply:
Will remove line 515

Reply:
Removed 515 and tested in TP/RP